lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -23,13 +25,17 @@ function multiplyFractions(node) {
     return Node.Status.noChange(node);
   }
   const atLeastOneFraction = node.args.some(
-    arg => Node.Type.isOperator(arg, '/'));
-  const hasPolynomialTerms = node.args.some(
-    arg => Node.PolynomialTerm.isPolynomialTerm(arg));
-  if (!atLeastOneFraction || hasPolynomialTerms) {
+      arg => Node.Type.isOperator(arg, '/'));
+
+  // If term is in the form of <constant fraction> <polynomial term>
+  // do not multiply in order to keep pedagogical style
+  // e.g 1/2 * x^2 -> 1/2 x^2 and not 1x^2 / 2
```

Comment:
this is a great comment! I'm really glad you have an example, too

do you think a more clear version of the example would be:

`e.g 1/2 * x^2 remains unchanged instead of stepping to 1*x^2 / 2`

cause this code doesn't do `1/2 * x^2 -> 1/2 x^2`